### PR TITLE
Story 23

### PR DESCRIPTION
**Pull request by limsammy**
Related Issue: #5 

Summary: Admin can add a new order

Author(s): Sam

### CODE REVIEW

File: app/views/admin/admin/show.html.erb [hunk 1]
```diff
@@ -19,8 +19,15 @@
             </div>
         </div>
     </div>
-
     <div class="control-panel-wrapper">
-        
+        <div class="row">
+            <div class="col-xs-6 col-md-4">
+                <%= link_to "Add New Item", new_admin_item_path, type: "button", class: "btn btn-success" %>
```
Comment (randy-darsh) [inline app/views/admin/admin/show.html.erb:25]: nice

File: app/views/admin/items/new.html.erb [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+
+    <%= f.label :category %>
+    <%= f.select :category_id, Category.all.collect { |c| [ c.title, c.id ] }, include_blank: false %>
+
+    <%= f.label :image %>
+    <%= f.text_field :image %>
+
+    <%= f.submit "Submit" %>
+  <% end %>
+</div>
```
Comment (amhursh) [inline app/views/admin/items/new.html.erb:21]: Cool. We can decide if we want to user paperclip with this later

File: config/routes.rb [hunk 3]
```diff
@@ -14,6 +14,7 @@
 
   namespace :admin do
     get '/dashboard', to: 'admin#show'
+    resources :items, only: [:new, :create]
```
Comment (amhursh) [inline config/routes.rb:17]: good to keep this nested.

File: app/views/admin/items/_form.html.erb [hunk 4]
```diff
@@ -0,0 +1,12 @@
...
+  <%= f.text_field :title %>
+
+  <%= f.label :description %>
+  <%= f.text_area :description %>
+
+  <%= f.label :price %>
+  <p>$</p><%= f.text_field :price %>
+
+  <%= f.select :category_id, Category.all.collect { |c| [ c.title, c.id ] }, include_blank: false %>
+<% end %>
```
Comment (randy-darsh) [inline app/views/admin/items/_form.html.erb:12]: dig the partials

File: spec/features/admin/admin_can_create_an_item_spec.rb [hunk 5]
```diff
@@ -0,0 +1,34 @@
...
+describe "Admin can add items" do
+  context "Admin visits /admin/items/new" do
+    scenario "Admin adds a new item for sale" do
+      admin = create(:user, role: 'admin')
+      category = create(:category, title: "Guitars")
+
+      visit login_path
+
+      fill_in "session[username]", with: admin.username
+      fill_in "session[password]", with: admin.password
```
Comment (amhursh) [inline spec/features/admin/admin_can_create_an_item_spec.rb:12]: You can use allow_any_instance_of to bypass the manual login

**Review by amhursh — APPROVED** [review]

**Pull request merged**